frontend/integration-tests/protractor.conf.ts

```diff
@@ -230,16 +246,19 @@ export const config = {
   },
 };
 
-export const checkLogs = async () =>
-  (
-    await browser
-      .manage()
-      .logs()
-      .get('browser')
-  ).map((log) => {
-    browserLogs.push(log);
-    return log;
-  });
+export const checkLogs = async () => {
+  if (config.capabilities.browserName === 'chrome') {
```

Comment:
Prefer an early return, which is easier to reason about.

```suggestion
  if (config.capabilities.browserName !== 'chrome') {
    return;
  }
  // ...capture logs...
```

Reply:
Sure, will update...